src/main/java/bio/terra/cli/serialization/userfacing/UFWorkspaceLight.java

```diff
@@ -92,6 +96,10 @@ public void print() {
     properties.forEach((key, value) -> OUT.println("  " + key + ": " + value));
     OUT.println("Created:           " + createdDate.format(DateTimeFormatter.ISO_LOCAL_DATE));
     OUT.println("Last updated:      " + lastUpdatedDate.format(DateTimeFormatter.ISO_LOCAL_DATE));
+    if (spendProfile == null) {
```

Comment:
Hmm, I saw that spend profile was optional in a WorkspaceDescription which is why I put the if. Maybe only for data collections?

@cbookg WDYT about displaying?

Reply:
In general I think I agree with not displaying the spend profile.  Though it would be handy to see that the correct profile was being used, even for early users.

Reply:
asked in pm/ux chat

Reply:
what is the conclusion of the chat?